index.js

```diff
@@ -77,6 +77,17 @@ module.exports = {
     }
   },
 
+  /**
+     Returns whether this app is using a module unification format.
+     @private
+     @method isModuleUnification
+     @return {Boolean} Whether this app is using a module unification format.
+  */
+  isAppModuleUnification() {
```

Comment:
Should be using `this.project.isModuleUnificatiob()` (no need for a new helper method AFAICT)

Reply:
@rwjblue Good call, I've made that change.,